webui/react/src/components/PageHeaderFoldable.tsx

```diff
@@ -38,6 +38,7 @@ const PageHeaderFoldable: React.FC<Props> = (
   if (options && options.length > 0) {
     if (options.length === 1) dropdownClasses.push(css.optionsDropdownOneChild);
     if (options.length === 2) dropdownClasses.push(css.optionsDropdownTwoChild);
+    if (options.length === 3) dropdownClasses.push(css.optionsDropdownThreeChild);
```

Comment:
It seems like there should be a better way to do this than all these if statements but it's not coming to me at the moment